cicd/pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,61 @@
+:_content-type: ASSEMBLY
+[id="customizing-configurations-in-the-tektonconfig-cr"]
+= Customizing configurations in the TektonConfig custom resource
+include::_attributes/common-attributes.adoc[]
+:context: customizing-configurations-in-the-tektonconfig-cr
+
+toc::[]
+
+[role="_abstract"]
+You can customize the following {pipelines-title} behaviors by using the `TektonConfig` custom resource (CR):
+
+* Configuring the {pipelines-title} control plane
+* Changing the default service account
+* Disabling the service monitor
+* Configuring the pipeline resolvers
+* Disabling cluster tasks and pipeline templates
+* Disabling the integration of Tekton Hub
+* Disabling the automatic creation of RBAC resources
+* Pruning of task runs and pipeline runs
+
+[discrete]
+== Prerequisites
+
+* You have installed the {pipelines-title} Operator.
+
+include::modules/op-configuring-pipelines-control-plane.adoc[leveloffset=+1]
+
+include::modules/op-configuration-options-with-default-values.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* See xref:../../cicd/pipelines/authenticating-pipelines-using-git-secret.adoc#op-configuring-ssh-authentication-for-git_authenticating-pipelines-using-git-secret[Configuring SSH authentication for Git] for more information about Git SSH secrets.
```

Comment:
@shipsing This belongs to the module, not an assembly, so it must be without an id. It is like this because of `xref`, which cannot be used in the module itself. see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections
Sentence: "Only use `. `formatting (`.Additional resources`) in a module or to follow a module in an assembly....."